pkg/address/validate_address.go

```diff
@@ -0,0 +1,53 @@
+package address
+
+import (
+	"fmt"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/zeta-chain/zetacore/pkg/chains"
+)
+
+// TODO : Use this function to validate Sender and Receiver address for CCTX
+// https://github.com/zeta-chain/node/issues/2697
+func ValidateAddressForChain(address string, chainID int64, additionalChains []chains.Chain) error {
```

Comment:
Same for this , Not using this function right now , but if used eventually it would make make the address validation a bit more strict . Wanted to get some thoughts before adding more unit tests and using this function